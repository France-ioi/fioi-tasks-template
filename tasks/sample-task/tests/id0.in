5 5
42 38
6
13 14
25 5
35 15
34 43
17 34
6 25
