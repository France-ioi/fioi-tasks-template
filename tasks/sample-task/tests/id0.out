3
5 5
25 5
35 15
42 38
